.travis.yml

```diff
@@ -41,6 +41,15 @@ script:
       -w "$(pwd)" \
       openjdk:8-jre \
       ./mvnw -B -pl core test -Dtest=*GenericContainerRuleTest
+  # Run Docker-in-Docker tests inside Alpine
+  - |
+      DOCKER_HOST=unix:///var/run/docker.sock DOCKER_TLS_VERIFY= docker run --rm \
+      -v "$HOME/.m2":/root/.m2/ \
+      -v /var/run/docker.sock:/var/run/docker.sock \
+      -v "$(pwd)":"$(pwd)" \
+      -w "$(pwd)" \
+      joepjoosten/openjdk-alpine-bash \
```

Comment:
We could remove this test as well, but I thought this might be a helpful regression test for alpine compatibility. The image itself is very simple however, so we could easily implement our own version.